blues/sparrow-starter
# feat: tablet responsive sparrow

# Problem Context

We're making the site tablet responsive before the beta launch next week, as well as handling displaying gateway name dynamically (Michelle had this removed from the sensor details page, btw) and fixing some CSS issues like the weird scrollbar.

## Changes

Leverage Ant Design columns and breakpoints to help make our site more responsive and less tied to either a desktop or tablet view and remove now unnecessary CSS.

Add locations to sensor cards, remove gateway location from sensor details page, remove drop shadow from cards to make them more alike with our latest sensor / gateway details, made charts and cards responsive based on viewport size, adjusted font sizes for different text on cards to give proper emphasis.

## Screenshot (if applicable)

![Screen Shot 2022-02-11 at 5 18 52 PM](https://user-images.githubusercontent.com/20400845/153678308-82d5f405-daf5-4788-b77c-06e8dee1abf7.png)

![Screen Shot 2022-02-11 at 5 19 05 PM](https://user-images.githubusercontent.com/20400845/153678310-3e8d2e98-1d9b-49d3-a785-49232cffd503.png)

## Testing

Unit / integration / e2e tests?

Unit and e2e tests were fixed up, and I made an idea in Aha to add more unit tests to our `pages/` folder as a separate story now that the majority of the refactoring under the hood is completed.

Steps to test manually?

1. Pull down the local branch and run using `yarn dev`
2. Using Chrome Dev Tools resize the screen to 1280 x 752 (this is our desktop version sized mockup)
2. Navigate around the site observing how clean, polished and slick everything looks
3. Resize the browser to 752 x 1280 (this is our tablet sized version)
4. Check out how the cards resize to stack and take up the majority of the screen - same for the charts on the sensor details page

Any other sorts of testing notes to include?

## Any other related PRs

n/a

## Ticket(s)

* https://www.pivotaltracker.com/story/show/181160023
* https://www.pivotaltracker.com/story/show/181210337
* https://www.pivotaltracker.com/story/show/181175185
